questionnaire/submissions.test.js

```diff
@@ -472,3 +489,46 @@ describe('Questionnaire submissions', () => {
         });
     });
 });
+
+describe('Questionnaire resubmissions', () => {
+    describe('Message Bus operational', () => {
+        it('should set a submission to IN_PROGRESS', async () => {
+            // eslint-disable-next-line global-require
+            const createQuestionnaireDAL = require('./questionnaire-dal');
+            const questionnaireDAL = createQuestionnaireDAL({logger: jest.fn()});
+            const createSubmissionsService = jest.requireActual('./submissions-service');
+            const submissionsService = createSubmissionsService({logger: jest.fn()});
+            await submissionsService.postFailedSubmissions();
+            expect(questionnaireDAL.updateQuestionnaireSubmissionStatus).toHaveBeenCalledWith(
+                '93259f6d-1826-4e97-ba39-53b4e232dd81',
+                'IN_PROGRESS'
+            );
+            expect(questionnaireDAL.updateQuestionnaireSubmissionStatus).not.toHaveBeenCalledWith(
+                '93259f6d-1826-4e97-ba39-53b4e232dd81',
+                'FAILED'
+            );
+        });
+    });
+    describe('Message Bus NOT operational', () => {
+        it('should set a submission to FAILED', async () => {
+            // eslint-disable-next-line global-require
+            const createQuestionnaireDAL = require('./questionnaire-dal');
+            const questionnaireDAL = createQuestionnaireDAL({logger: jest.fn()});
+            const createSubmissionsService = jest.requireActual('./submissions-service');
+            const submissionsService = createSubmissionsService({logger: jest.fn()});
+            await submissionsService.postFailedSubmissions();
```

Comment:
This test isn't independent. These lines could be removed and it would still pass due to the previous test